.editorconfig

```diff
@@ -39,7 +39,7 @@ dotnet_style_predefined_type_for_member_access = true:suggestion
 # Require var all the time.
 csharp_style_var_for_built_in_types = true:suggestion
 csharp_style_var_when_type_is_apparent = true:suggestion
-csharp_style_var_elsewhere = false
+csharp_style_var_elsewhere = false:suggestion
```

Comment:
Does this make suggestion to change `var` into explicit type?

Reply:
Yes, for not built in types and when types are not visible in the same line